week_1/GymWebPage.css

```diff
@@ -4,16 +4,39 @@
     font-optical-sizing: auto;
     font-style: normal;
 }
+html{
+    overflow-x: hidden;
+    background:linear-gradient(135deg, #ffffff 0%,#4d5154 100%);
+}
+section{
+    padding-bottom: 300px;
```

Comment:
@codal-uchoudhary 
Suggestion - Check how we can apply padding in one line. Like top-bottom and left-right